### ISSUE TITLE
Inclusion of Bias in Lab 1, Part 1

### ISSUE THREAD

**Issue by nachiketdhamankar** [aamini/introtodeeplearning#31]
In the below code, I can see the activation function and the weights, but where exactly do we add the bias?

~~~~
# Import relevant packages
from tensorflow.keras import Sequential
from tensorflow.keras.layers import Dense

# Define the number of inputs and outputs
n_input_nodes = 2
n_output_nodes = 3

# First define the model 
model = Sequential()


'''TODO: Define a dense (fully connected) layer to compute z'''
# Remember: dense layers are defined by the parameters W and b!
# You can read more about the initialization of W and b in the TF documentation :) 
dense_layer = Dense(n_output_nodes, input_shape=(n_input_nodes,),activation='sigmoid')

# Add the dense layer to the model
model.add(dense_layer)
~~~~

**Comment by Tirthnp** [comment]
I have the same question

**Comment by aamini** [comment]
The bias is included in the `tf.keras.layers.Dense` layer by default. See https://www.tensorflow.org/api_docs/python/tf/keras/layers/Dense.
